### PR TITLE
INFRA-66 Update dependency to point to latest upstream release

### PR DESCRIPTION
**Pull request by garethr**
As an experiment, for this release, we are using the Puppet Strings
project to provide class and type reference documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,426 +297,9 @@ Build a cluster of hosts running Docker Swarm configured by Puppet.
...
-
-*Required* The private registry password for the specified user.
-
-#####`email`
-
-*Required* The email address associated with the private registry
-account.
+[Full API reference
+documentation](http://puppetlabs.github.io/puppetlabs-docker_platform/)
+is available as generated by Puppet Strings.
```
Comment (jbondpdx) [inline README.md]: Maybe we should link to Strings where you mention it? I'll leave it up to you; just throwing it out there.

**Comment by DavidS** [comment]
strings reference +:100:

**Comment by jbondpdx** [comment]
Another question: how will users behind a firewall access the types information? That is, is there a way for users to get the Strings generated docs in terminal?

**Comment by garethr** [comment]
@DavidS @jbondpdx I've added a link to Strings and instructions for building the documentation yourself.

**Comment by jbondpdx** [comment]
+1

**Pull request merged**